presto-hive/src/test/java/io/prestosql/plugin/hive/TestOrcPageSourceMemoryTracking.java

```diff
@@ -194,7 +194,13 @@ private void testPageSource(boolean useCache)
         FileFormatDataSourceStats stats = new FileFormatDataSourceStats();
         ConnectorPageSource pageSource = testPreparer.newPageSource(stats, useCache ? CACHED_SESSION : UNCACHED_SESSION);
 
-        assertEquals(pageSource.getSystemMemoryUsage(), 0);
+        if (useCache) {
```

Comment:
Should these be in the previous commit *Add memory tests for both cached and uncached ORC*?

Reply:
I have no idea 🤷